packages/KTable/KTable.vue

```diff
@@ -52,6 +52,15 @@
               appearance="primary"
               @click="$emit('ktable-empty-state-cta-clicked')"
             >
+              <template
+                v-slot:icon
+                v-if="emptyStateActionButtonIcon">
+                <KIcon
+                  :icon="emptyStateActionButtonIcon"
+                  color="white"
```

Comment:
Non-blocking question: will we always want `white` for the icon color? Hypothetically if for some reason we wanted an icon in a button with the outline style, I don't think the icon would be visible. Possibly an unreasonable edge-case but just wanted to surface it.